hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -307,9 +309,29 @@ public OMClientResponse handleWriteRequest(OMRequest omRequest,
     OMClientResponse omClientResponse = null;
     omClientRequest =
         OzoneManagerRatisUtils.createClientRequest(omRequest, impl);
-    omClientResponse = omClientRequest
-        .validateAndUpdateCache(getOzoneManager(), transactionLogIndex,
-            ozoneManagerDoubleBuffer::add);
+    try {
+      omClientResponse = omClientRequest
+          .validateAndUpdateCache(getOzoneManager(), transactionLogIndex,
+              ozoneManagerDoubleBuffer::add);
+    } catch (InvalidPathException e) {
+      LOG.error("Got InvalidPathException while handling request: {}",
```

Comment:
I don't think we should log to OM log client-induced errors. This could be something logged to the audit log instead for a failed request.

Reply:
@kerneltime Thank you for the review. Updated the PR, please have a look.